hummingbot/client/config/client_config_map.py

```diff
@@ -0,0 +1,851 @@
+            prompt=lambda cm: f"Select the desired telegram mode ({'/'.join(list(TELEGRAM_MODES.keys()))})"
+        )
+    )
+    send_error_logs: bool = Field(
+        default=True,
+        description="Error log sharing",
+        client_data=ClientFieldData(
+            prompt=lambda cm: "Would you like to send error logs to hummingbot? (Yes/No)",
+        ),
+    )
+    previous_strategy: Optional[str] = Field(
+        default=None,
+        description="Can store the previous strategy ran for quick retrieval."
+    )
+    db_mode: Union[tuple(DB_MODES.values())] = Field(
+        default=DBSqliteMode(),
+        description=("Advanced database options, currently supports SQLAlchemy's included dialects"
+                     "\nReference: https://docs.sqlalchemy.org/en/13/dialects/"
+                     "\nTo use an instance of SQLite DB the required configuration is \n  db_engine: sqlite"
+                     "\nTo use a DBMS the required configuration is"
+                     "\n  db_host: 127.0.0.1\n  db_port: 3306\n  db_username: username\n  db_password: password"
+                     "\n  db_name: dbname"),
+        client_data=ClientFieldData(
+            prompt=lambda cm: f"Select the desired db mode ({'/'.join(list(DB_MODES.keys()))})",
+        ),
+    )
+    pmm_script_mode: Union[tuple(PMM_SCRIPT_MODES.values())] = Field(
+        default=PMMScriptDisabledMode(),
+        client_data=ClientFieldData(
+            prompt=lambda cm: f"Select the desired PMM script mode ({'/'.join(list(PMM_SCRIPT_MODES.keys()))})",
+        ),
+    )
+    balance_asset_limit: Dict[str, Dict[str, Decimal]] = Field(
+        default={exchange: {} for exchange in AllConnectorSettings.get_exchange_names()},
+        description=("Balance Limit Configurations"
+                     "\ne.g. Setting USDT and BTC limits on Binance."
+                     "\nbalance_asset_limit:"
+                     "\n  binance:"
+                     "\n    BTC: 0.1"
+                     "\n    USDT: 1000"),
+        client_data=ClientFieldData(
+            prompt=lambda cm: (
+                "Use the `balance limit` command e.g. balance limit [EXCHANGE] [ASSET] [AMOUNT]"
+            ),
+        ),
+    )
+    manual_gas_price: Decimal = Field(
+        default=Decimal("50"),
+        description="Fixed gas price (in Gwei) for Ethereum transactions",
+        gt=Decimal("0"),
+        client_data=ClientFieldData(
+            prompt=lambda cm: "Enter fixed gas price (in Gwei) you want to use for Ethereum transactions",
+        ),
+    )
+    gateway: GatewayConfigMap = Field(
+        default=GatewayConfigMap(),
+        description=("Gateway API Configurations"
+                     "\ndefault host to only use localhost"
+                     "\nPort need to match the final installation port for Gateway"),
+    )
+    anonymized_metrics_mode: Union[tuple(METRICS_MODES.values())] = Field(
+        default=AnonymizedMetricsDisabledMode(),
```

Comment:
This default value is incorrect. The current default value for this field is metrics enabled.

Reply:
Addressed